lib/inc/drogon/MultiPart.h

```diff
@@ -174,7 +173,7 @@ class DROGON_EXPORT MultiPartParser
 
   protected:
     std::vector<HttpFile> files_;
-    std::map<std::string, std::string> parameters_;
+    std::unordered_map<std::string, std::string> parameters_;
```

Comment:
please use std::unordered_map<std::string,
                             std::string,
                             utils::internal::SafeStringHash> for consistency.